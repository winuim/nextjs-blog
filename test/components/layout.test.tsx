import React from 'react';
import {render} from '../testUtils';
import Layout from '../../components/layout';

describe('Layout component', () => {
  it('matches snapshot', () => {
    const {asFragment} = render(
      <Layout>
        <span>test</span>
      </Layout>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('matches snapshot, home', () => {
    const {asFragment} = render(
      <Layout home>
        <span>test</span>
      </Layout>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
