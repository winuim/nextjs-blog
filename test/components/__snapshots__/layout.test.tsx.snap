// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <a
        href="/"
      >
        <img
          alt="[Your Name]"
          class="headerImage borderCircle"
          src="/images/profile.jpg"
        />
      </a>
      <h2
        class="headingLg"
      >
        <a
          class="colorInherit"
          href="/"
        >
          [Your Name]
        </a>
      </h2>
    </header>
    <main>
      <span>
        test
      </span>
    </main>
    <div
      class="backToHome"
    >
      <a
        href="/"
      >
        ‚Üê Back to home
      </a>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Layout component matches snapshot, home 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <img
        alt="[Your Name]"
        class="headerHomeImage borderCircle"
        src="/images/profile.jpg"
      />
      <h1
        class="heading2Xl"
      >
        [Your Name]
      </h1>
    </header>
    <main>
      <span>
        test
      </span>
    </main>
  </div>
</DocumentFragment>
`;
